spec.html

```diff
@@ -34532,8 +34534,203 @@ <h2>Syntax</h2>
           [+UnicodeMode] `-`
           CharacterClassEscape[?UnicodeMode]
           CharacterEscape[?UnicodeMode]
+
+        ClassContents ::
+          ClassUnion
+          ClassIntersection
+          ClassSubtraction
+
+        ClassUnion ::
+          ClassRange ClassUnion?
+          ClassOperand ClassUnion?
+
+        ClassIntersection ::
+          ClassOperand `&amp;&amp;` [lookahead != `&amp;`] ClassOperand
+          ClassIntersection `&amp;&amp;` [lookahead != `&amp;`] ClassOperand
+
+        ClassSubtraction ::
+          ClassOperand `--` ClassOperand
+          ClassSubtraction `--` ClassOperand
```

Comment:
There is good reason to have subtraction on the left, so that A -- B -- C is grouped as ((A -- B) -- C). 

It doesn't matter with associative operators like &&.

Reply:
I see @gibson042’s 👍  on @macchiati’s comment. Does this mean we can resolve this conversation, or are any changes needed?